cypress/integration/ete/new_account_review.3.cy.ts

```diff
@@ -168,6 +168,11 @@ describe('New account review page', () => {
 });
 
 describe('New account newsletters page', () => {
+	beforeEach(() => {
+		cy.intercept('GET', 'https://m.code.dev-theguardian.com/**', (req) => {
+			req.reply(200);
+		});
+	});
```

Comment:
this fixes an unrelated issue which started occurring in other tests, do to an error on the page it redirected to

since we just want to check the redirect happened, this was added in